pkg/controller/kibana/controller.go

```diff
@@ -65,13 +66,18 @@ var (
 func (r *ReconcileKibana) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	es, err := k8shandler.GetElasticsearchCR(r.client, request.Namespace)
 	if err != nil {
-		return reconcileResult, fmt.Errorf("skipping kibana reconciliation in %q: %s", request.Namespace, err)
+		logrus.Infof("skipping kibana reconciliation in %q: %s", request.Namespace, err)
+		return reconcileResult, nil
```

Comment:
Wouldn't inlining `reconcile.Result{RequeueAfter: reconcilePeriod}` make the case more readable?

Reply:
updated.